invenio_app_rdm/requests_ui/views/requests.py

```diff
@@ -103,9 +103,10 @@ def user_dashboard_request_view(request, **kwargs):
     is_draft_submission = request_type == CommunitySubmission.type_id
     is_record_inclusion = request_type == CommunityInclusion.type_id
     is_member_invitation = request_type == CommunityInvitation.type_id
+    is_legacy_record_upgrade = request_type == "legacy-record-upgrade"
```

Comment:
we should discuss this with the team how to avoid it, not ideal for invenio-app-rdm to know about legacy record upgrade type

Reply:
+1 on this, if legacy record is something zenodo-specific then it feels better to either have it on zenodo or make it a broader, more generic feature if it's relevant for other instances as well